pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/rook/rook/pkg/operator/ceph/config/keyring"
 	opspec "github.com/rook/rook/pkg/operator/ceph/spec"
 	"github.com/rook/rook/pkg/operator/k8sutil"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
There should be no need to specify this as `v1` since the module is already called `v1`

Reply:
Yes, it's annoying vscode keeps adding it...

Reply:
Annoying, but this won't break anything, so I'm fine keeping it.